WalletWasabi.Fluent.Desktop/Program.cs

```diff
@@ -192,22 +205,22 @@ private static void CurrentDomain_UnhandledException(object? sender, UnhandledEx
 		}
 	}
 
-	// This is required to bootstrap Avalonia's Visual Previewer
+	[SuppressMessage("CodeQuality", "IDE0051:Remove unused private members", Justification = "Required to bootstrap Avalonia's Visual Previewer")]
 	private static AppBuilder BuildAvaloniaApp()
 	{
-		return BuildAvaloniaApp(false);
+		return BuildAvaloniaApp(false, null!);
 	}
 
 	// Avalonia configuration, don't remove
-	private static AppBuilder BuildAvaloniaApp(bool startInBg)
+	private static AppBuilder BuildAvaloniaApp(bool startInBg, TerminateService terminateService)
 	{
 		bool useGpuLinux = true;
 
 		var result = AppBuilder.Configure(() => new App(async () =>
 			{
-				if (Global is { } global)
+				if (Global is { } global && terminateService is not null)
```

Comment:
`terminateService` is not nullable but we still check whether it is or not. 

As far as I can see the source of the issue is the designer again, I did another reordering [in this commit](https://github.com/zkSNACKs/WalletWasabi/commit/61e1fc6830c474983dda1f9c957231a8eb59a1b4).
Maybe it can be further improved, but I think it is a good move. Neither `Global` nor `terminateService` are not needed for the designer.

Opinions? @kiminuo 
@SuperJMN Could you check if the designer is still working with this commit? I have 222 Rider and the designer is not compatible with it yet. 
@danwalmsley Any other thing that can be broken with my commit?

Reply:
@soosr I've just checked the most recent version of this PR and the Previewer works as expected.

![image](https://user-images.githubusercontent.com/3109851/171297873-dd2e70f5-ef38-4db6-9717-40b94f936296.png)

![image](https://user-images.githubusercontent.com/3109851/171297920-9f67b763-4b9b-49cb-8192-141051f0c7ae.png)

![image](https://user-images.githubusercontent.com/3109851/171297942-a4b585c3-2c16-4254-b38d-62ac4f6f8d2f.png)

Reply:
> @soosr I've just checked the most recent version of this PR and the Previewer works as expected.

The modification hasn't been applied to this branch yet. Could you test [this branch](https://github.com/soosr/WalletWasabi/tree/further-improvement) instead? @SuperJMN